scss/templates/_featured.scss

```diff
@@ -0,0 +1,8 @@
+.rev-Featured {
+  p {
+    font-size: $font-size-small;
+  }
+}
+.rev-Featured--center {
```

Comment:
wasn't sure if this was the best way to handle whether the featured text was centered or not. I figured if they had bullet points as the supporting text, then you might not want it centered.

Reply:
talked to laura and decided to just use a util class for this instead `Text-center`